pkg/controller/installation/products/launcher/asserts_test.go

```diff
@@ -50,17 +49,6 @@ func assertInstallationSuccessfullyReconciled(scenario LauncherTestScenario, con
 		return errors.New(fmt.Sprintf("%s ConsoleURL not equal MasterURL ", defaultLauncherName))
 	}
 
-	// RHSSO integration should be configured.
-	rhssoConfig, err := configManager.ReadRHSSO()
```

Comment:
What is the reason for this being removed?

Reply:
The keycloak realm is no longer modified, so this is no longer needed.

Reply:
Thanks @pb82